.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Ion-test-driver issue.
+assignee: cheqianh
```

Comment:
Don't set the assignee to yourself. Can you set it to `{{ payload.sender.login }}` so that the person who created the commit will get assigned?